eth2util/registration/registration.go

```diff
@@ -77,13 +81,13 @@ func getRegistrationDomain() (eth2p0.Domain, error) {
 }
 
 // GetMessageSigningRoot returns the validator registration message signing root created by the provided parameters.
-func GetMessageSigningRoot(msg *eth2v1.ValidatorRegistration) ([32]byte, error) {
+func GetMessageSigningRoot(msg *eth2v1.ValidatorRegistration, forkVersion []byte) ([32]byte, error) {
```

Comment:
root cause, we implemented signing logic without proper testing. We have existing code that generates valid registrations, please write a test that ensures this code and that code generates the same sigs

Reply:
done